templates/environment.template.json

```diff
@@ -94,6 +94,20 @@
         "description": "The number of internal subnets to provision."
       }
     },
+    "firewallsNsgName": {
+      "type": "string",
+      "defaultValue": "Disabled",
+      "metadata": {
+        "description": "Name of the Resource Group where the firewall NSG resides, used for the resourceId function to get the firewall NSG resource"
```

Comment:
Should this description be with the `firewallsResourceGroupName` parameter, and then the description for that one here?